homeassistant/components/victron_gx_mqtt/strings.json

```
@@ -0,0 +1,1335 @@
+        "name": "Load power"
+      },
+      "acload_power_phase": {
+        "name": "Load power on {phase}"
+      },
+      "acload_voltage": {
+        "name": "Load voltage"
+      },
+      "acload_voltage_phase": {
+        "name": "Load voltage on {phase}"
+      },
+      "alternator_charge_current_limit": {
+        "name": "Charge current limit"
+      },
+      "alternator_dc_current": {
+        "name": "DC output current"
+      },
+      "alternator_dc_power": {
+        "name": "DC output power"
+      },
+      "alternator_dc_voltage": {
+        "name": "DC output voltage"
+      },
+      "alternator_input_current": {
+        "name": "Input current"
+      },
+      "alternator_input_power": {
+        "name": "Input power"
+      },
+      "alternator_input_voltage": {
+        "name": "Input voltage"
+      },
+      "alternator_state": {
+        "name": "Alternator state"
+      },
+      "auxiliary_battery_voltage": {
+        "name": "Auxiliary battery voltage"
+      },
+      "battery_automatic_syncs": {
+        "name": "Automatic syncs"
+      },
+      "battery_capacity": {
+        "name": "Battery capacity"
+      },
+      "battery_cell_cell_id_voltage": {
+        "name": "Battery cell {cell_id} voltage"
+      },
+      "battery_cell_imbalance": {
+        "name": "Battery cell imbalance"
+      },
+      "battery_cell_voltage_deviation": {
+        "name": "Battery cell voltage deviation"
+      },
+      "battery_charged_energy": {
+        "name": "Charged energy"
+      },
+      "battery_consumed_amphours": {
+        "name": "Battery consumed amp-hours"
+      },
+      "battery_current": {
+        "name": "DC Bus current"
+      },
+      "battery_discharged_energy": {
+        "name": "Discharged energy"
+      },
+      "battery_high_charge_current": {
+        "name": "Battery high charge current"
+      },
+      "battery_high_charge_temperature": {
+        "name": "Battery high charge temperature"
+      },
+      "battery_high_discharge_current": {
+        "name": "Battery high discharge current"
+      },
+      "battery_installed_capacity": {
+        "name": "Battery installed capacity"
+      },
+      "battery_internal_failure": {
+        "name": "Battery internal failure"
+      },
+      "battery_low_cell_voltage": {
+        "name": "Battery low cell voltage"
+      },
+      "battery_low_charge_temperature": {
+        "name": "Battery low charge temperature"
+      },
+      "battery_max_cell_temperature": {
+        "name": "Battery maximum cell temperature"
+      },
+      "battery_max_cell_voltage": {
+        "name": "Battery maximum cell voltage"
+      },
+      "battery_max_charge_current": {
+        "name": "Battery maximum charge current allowed"
+      },
+      "battery_max_charge_voltage": {
+        "name": "Battery maximum charging voltage allowed"
+      },
+      "battery_max_discharge_current": {
+        "name": "Battery maximum discharge current allowed"
```

I assume these are values for the "allowed current" or "allowed voltage". As such these words need be compounded. Otherwise it sounds like a binary sensor or switch:

```suggestion
      "battery_max_charge_current": {
        "name": "Battery maximum allowed charge current"
      },
      "battery_max_charge_voltage": {
        "name": "Battery maximum allowed charging voltage"
      },
      "battery_max_discharge_current": {
        "name": "Battery maximum allowed discharge current"
```